src/app-view.js

```diff
@@ -30,6 +31,10 @@ class AppView extends React.Component {
         this.checkCurrentSection();
         this.checkDisplayedComponent();
 
+        if (this.checkMobileDevice()) {
+            this.setSidebarVisibility();
+        }
+
         window.addEventListener('keyup', this.handleKeyDown);
```

Comment:
should we not switch off listener  for mobile devices?

Reply:
Definitely!